src/js/components/WidgetExample.jsx

```diff
@@ -15,7 +15,7 @@ const WidgetExample = ({name, keywords, children, setShow}) => {
 	}	
 	if (setShow) setShow(true);
 	
-	return (<Card body>
+	return (<Card className={className} name={name.replace(/[^a-zA-Z0-9-_]/g, "").toLowerCase()} body>
```

Comment:
Please add a short comment on what the name and canonicalisation are for. Thanks.

Reply:
Will do